docs/Guidelines/Brand/brand.stories.mdx

```diff
@@ -0,0 +1,125 @@
+events with our own technology to create a fair,
+honest and safe place for people to resell their
+tickets online. By placing a selling-limit of no
+more than 20% above the original ticket price,
+we aim to stop profiteering, fraud, keep ticket
+reselling fair, and help those fans who want to go
+to an event without getting ripped off.
+
+We started TicketSwap with simplicity in mind.
+Our platform will remain as simple as possible
+for anyone to enter the site, buy sell and place
+wanted listings for second-hand tickets.
+
+Our business values will remain being customer-
+driven, honest, safe, transparent, and simple.
+
+The journey we are on is not only to pursue
+technological advances and business growth:
+we want to change the world. By this we mean
+we will stick to our values in order to change
+the way the world sees the secondary ticketing
+industry: we will bring fairness to the market,
+and we will continue to lobby policy-makers,
+sit with event organizers, engage with artists
+and preach the gospel of ticketing to the fans.
+After all, we are fans too.
+Sound ambitious? Always.
+We will never rest until the world knows
+TicketSwap is here for you, the fans.
+
+## Vision
+
+We’re in business to make the lives of fans better.
+
+## Design principles
+
+## Tone of voice
+
+The way we are positioned, shows the world
+exactly why we are different than others.
+TicketSwap’s mission is to be the safest and
+fairest way to buy and sell secondhand tickets.
+TicketSwap believes the secondary ticket market
+should be about providing fans with a safe way
+to recoup the money they’ve paid for tickets they
+can no longer use. In its current state, real fans
+aren’t getting a fair shot at buying tickets and
+being priced out of attending events altogether.
+
+TicketSwap’s stance is simple and focuses on
+empowering consumers: the best way to beat touts
+is to create an alternative that is fairer, safer and
+more transparent. TicketSwap carefully chooses its
+partners and is always open for collaborations that
+could make the online ticketing world safer.
+You’ll find this state of mind in the employees that
+work at our head office in Amsterdam, or remote
+in different countries. We all have a clear vision of
+how real fans should be treated, because we’ve all
+been in that situation where there was room for a
+lot of improvement. It’s why we tackled it and made
+it our priority.
+
+We believe that our communication should be
+on the same level as those who we’re speaking
+with, no matter what language they use.
+Transparency should be at the core of all our
+messaging.
+
+Our voice is our users voice and it’s key in our
+social media efforts. We try to stay relevant with a
+hint of cheekiness, participating in trending topics
+with our own unique take on them.
+The tone of voice in all communication is informal,
+but correct. When situations require you to be
+formal, that is not a problem.
+
+## Terminology
```

Comment:
Don't think we should add empty blocks. It looks kinda messy imo and it looks "not done", which is not something we want in main right?

Reply:
I'll remove them for now